### PR TITLE
Add Socratic 32

### PR DESCRIPTION
**Pull request by stickies-v**
Topics and links can be pushed directly to this branch, or added as comments/review.

### CODE REVIEW

**Comment by ConorOkus** [comment]
**Misc / General Stuff** 
- First Mobile Native Hardware Wallet, https://twenty-two.xyz/

**UX/Design**
- Bitcoin Symbol Campaign, https://twitter.com/Nexo/status/1767566233384186180

**Lightning/LDK**
- Unleashing Liquidity on the Lightning Network with "lightning-liquidity", https://lightningdevkit.org/blog/unleashing-liquidity-on-the-lightning-network-with-lightning-liquidity/

File: _posts/2024-03-13-socratic-seminar-32.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- [Re enable `OP_CAT`](https://github.com/bitcoin-inquisition/bitcoin/pull/39)
+
+### [BIPs](https://github.com/bitcoin/bips)
+- [Add OP\_VAULT (BIP 345)](https://github.com/bitcoin/bips/pull/1421)
+
+### [BOLT]
+-
+
+## New Releases
+- [Bitcoin Core 26.1 Release Candidates](https://bitcoincore.org/bin/bitcoin-core-26.0/)
```
Comment (glozow) [inline _posts/2024-03-13-socratic-seminar-32.md:89]:
```suggestion
- [Bitcoin Core 26.1 Release Candidates](https://bitcoincore.org/bin/bitcoin-core-26.1/)
```

Comment (glozow) [reply _posts/2024-03-13-socratic-seminar-32.md]: fixed in a push

**Comment by stickies-v** [comment]
Preview looks good, ready for (squash) merge I think?

File: _posts/2024-03-13-socratic-seminar-32.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+-
+
+## Miscellaneous
+- [Marathon Digital Holdings Launches Slipstream](https://ir.mara.com/news-events/press-releases/detail/1343/marathon-digital-holdings-launches-slipstream)
+- [Sharing the Code Behind Bitkey](https://bitkey.build/sharing-the-code-behind-bitkey/)
+- [Swaproot: cheaper and more private on-chain deposits on Phoenix](https://acinq.co/blog/phoenix-swaproot)
+- [Schnorr (Incremental) Half Aggregation #261](https://github.com/BlockstreamResearch/secp256k1-zkp/pull/261)
+- [New version of http://learnmeabitcoin.com is now online](https://twitter.com/in3rsha/status/1765727225918128208)
+- [Introducing Portal: the hardware wallet for mobile users](https://twenty-two.xyz/)
+- [Bitcoin Symbol Campaign](https://github.com/londonbitdevs/londonbitdevs-website/pull/26)
```
[the remark this answers is not in the document]
Comment (stickies-v) [reply _posts/2024-03-13-socratic-seminar-32.md]: Mea culpa. ty

**Pull request merged**